src/Core/Components/Overflow/FluentOverflow.razor.cs

```diff
@@ -24,6 +25,7 @@ public partial class FluentOverflow : FluentComponentBase, IAsyncDisposable
 
     /// <summary />
     protected string? StyleValue => new StyleBuilder(Style)
+        .AddStyle("visibility",_visibility)
```

Comment:
Probably better to use `.AddStyle("visibility", "hidden", when: VisibleOnLoad == true)`.
So, you can remove the `_visibility` variable and the style will be applied only when the property is True (and the user could override the style in other cases).